fcrepo-transform/src/main/java/org/fcrepo/transform/TransformationFactory.java

```diff
@@ -56,7 +59,7 @@ public TransformationFactory() {
     public <T> Transformation<T> getTransform(final MediaType contentType, final InputStream inputStream) {
         final String mimeType = contentType.toString();
         if (mimeToTransform.containsKey(mimeType)) {
-            return (Transformation<T>) mimeToTransform.get(contentType.toString()).newTransform(inputStream);
+            return mimeToTransform.get(contentType.toString()).apply(inputStream);
```

Comment:
No, it doesn't. The tests wouldn't even pass if it did that. `RdfStream` has no relevance here. Look at the new type of `mimeToTransform`: `Map<String, Function<InputStream, Transformation>>`. Give it a `String`, it will give you a `Function<InputStream, Transformation>`. The old type let you give it a `String` and gave you a `Transform` which was only used to produce a new `Transform` via `newTransform(InputStream)` (which no longer exists). `newTransform(InputStream)` was semantically equivalent to `Function<InputStream, Transformation>`, but the new way is not idiosyncratic. The semantics are the same.